scripts/uncache.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+// NB: loads .env file into respective env vars
+// (see: https://github.com/motdotla/dotenv#readme)
+const dotenv = require('dotenv')
+
+const {argv, env, exit} = require('process');
+const {execSync} = require('child_process');
+
+const {loadEnv} = require('./common');
+
+const stage = argv[2];
+if (!stage) {
+    console.error("Usage: uncache.js <stage name>")
+    exit(1);
+}
+
+const envPath = `.${stage}.env`;
+loadEnv(envPath);
+
+const dist_id = env['CDF_DISTRIBUTION_ID'];
```

Comment:
Same comment as above. Some docs in the readme on how to find the value and populate the env file may be useful.